src/Bookmarks/index.jsx

```diff
@@ -1,28 +1,58 @@
-import React from "react";
-import { useDispatch } from "react-redux";
+import { useDispatch, useSelector } from "react-redux";
 import PropTypes from "prop-types";
 import { BaniList, actions, StatusBarComponent, SafeArea, useTheme } from "@common";
 import useBookmarks from "./hooks/useBookmarks";
 import useHeader from "./hooks/useHeader";
+import constant from "../common/constant";
 
 const Bookmarks = ({ navigation, route }) => {
   useHeader(navigation);
   const { theme } = useTheme();
   const { bookmarksData } = useBookmarks(route);
   const dispatch = useDispatch();
 
+  const fontFace = useSelector((state) => state.fontFace);
+
   const onPress = (item) => {
     dispatch(actions.setBookmarkPosition(item.item.shabadID));
     navigation.goBack();
   };
 
+  const formattedData = bookmarksData?.map((item, index) => {
```

Comment:
This can be reorganized slightly for readability. Destructure the `item` to get all the data that's required for this function first, and then use that later.

And also using ?? operator 

```js
const formattedData = bookmarksData?.map((item, index) => {
  const {
    gurmukhi,
    gurmukhiUni,
    tukGurmukhi,
    tukGurmukhiUni,
    translit,
    shabadID
  } = item;

  const title = isBaloo && gurmukhiUni ? gurmukhiUni : gurmukhi;

  const subtitle =
    isBaloo && tukGurmukhiUni
      ? tukGurmukhiUni
      : tukGurmukhi ?? translit ?? "";

  return {
    ...item,
    key: (shabadID ?? index).toString(),
    gurmukhi: title,
    tukGurmukhi: subtitle
  };
});

```